File: C++/Iniciante/1002.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>

int main() {
  
  double pi = 3.14159, area, raio;
  std::cin >> raio;

  area = pi * std::pow(raio, 2);

  printf("A=%.4lf\n", area);
  
}

File: JavaScript/Iniciante/1002.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

n = 3.14159
raio = parseFloat(lines.shift())
area = n * Math.pow(raio, 2)

console.log(`A=${area.toFixed(4)}`)
